.env.example

```diff
@@ -1,3 +1,3 @@
 RPC_URL=https://eth-mainnet.alchemyapi.io/v2/<API-KEY>
-BLOCK_NUMBER=15091338
+BLOCK_NUMBER=15257988
```

Comment:
Just a random thought but I wonder how we can automate this. Like maybe you have to specify that you want a fixed block number otherwise on every PR it'll update to the latest block or something

Reply:
Is it strictly necessary to update it though? I just updated it so that whoever else runs the tests forks at the same mainnet height as me. If we automate it, other folks might test using a different mainnet fork than I intended. It's sort of annoying especially for these gov related proposals where the vote count can change and the simulated gov process can fail in the tests.

But besides that it's easy enough to update it as part of the pull request. We can have a github action that hits Alchemy's getLatestblocknumber endpoint and populate that field.